exporter/influxdbexporter/exporter.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package influxdbexporter
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/influxdata/influxdb-observability/otel2influx"
+	otlpcollectorlogs "github.com/influxdata/influxdb-observability/otlp/collector/logs/v1"
+	otlpcollectormetrics "github.com/influxdata/influxdb-observability/otlp/collector/metrics/v1"
+	otlpcollectortrace "github.com/influxdata/influxdb-observability/otlp/collector/trace/v1"
+	otlplogs "github.com/influxdata/influxdb-observability/otlp/logs/v1"
+	otlpmetrics "github.com/influxdata/influxdb-observability/otlp/metrics/v1"
+	otlptrace "github.com/influxdata/influxdb-observability/otlp/trace/v1"
```

Comment:
Does this result in double registration of messages? AFAIK this should result in a warning at startup since the registry of messages is global: https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
This is probably a blocker since Protobufs say it may become a panic in a future version. I am blocking this for now until we clarify.
A workaround is to delete this and make otlpcollectorlogs accept serialized data for conversion instead of in-memory Protobuf messages.

Reply:
There was a double registration failure, and it is solved now.